docs/en/table_design/indexes/indexes_overview.md

```diff
@@ -0,0 +1,150 @@
+- StarRocks also allows users to manually create indexes, including Bitmap indexes and Bloom filter indexes.
+
+## Built-in indexes
+
+### Prefix indexes
+
+#### Description
+
+Specify one or more columns to comprise the sort key at table creation. The data rows in the table will be sorted based on the sort key and then stored on the disk.
+
+**During data writing, the Prefix index is automatically generated. After the data is sorted according to the specified sort key, every 1024 rows of data are included in one logical data block. An index entry that consists of the values of sort key columns  of the first row data in that logical data block is added to the Prefix index table.**
+
+With these two layers of sorting structures, queries can use binary search to quickly skip data that does not meet the query conditions, and can also avoid additional sorting operations during queries.
+
+:::tip
+
+The Prefix index is a sparse index, and its size is at least 1024 times smaller than the data volume. Therefore, it can generally be fully cached in memory, to accelerate query performance.
+
+:::
+
+#### Usage notes
+
+For a Duplicate Key table, sort key columns are specified in `DUPLICATE KEY`. For an Aggregate table or a Unique Key table, sort key columns are coupled with constrainted columns, and are specified in `AGGREGATE KEY` or `UNIQUE KEY`. From v3.0 onwards, a Primary Key table decouples sort key columns and primary key columns, providing more flexible capabilities. The sort key columns are specified in `ORDER BY`, and primary key columns are specified in `PRIMARY KEY`.
+
+The following example uses a Unique Key table to illustrate how to specify sort key columns at table creation and how they comprise a Prefix index.
+
+When creating the Unique Key table, you can specify sort key columns as `uid` and `name` in `DUPLICATE KEY`.
+
+```SQL
+CREATE TABLE user_access (
+    uid int,
+    name varchar(64),
+    age int, 
+    phone varchar(16),
+    last_access datetime,
+    credits double
+)
+DUPLICATE KEY(uid, name);
+```
+
+:::note
+
+After creating a Unique Key table, Aggregate table, or Unique Key table, you can use `DESCRIBE <table_name>;` to view its sort key columns. In the returned result, the columns whose `Key` field shows `true` are sort key columns. After creating a Primary Key table, you can use `SHOW CREATE TABLE <table_name>;` to view its sort key columns.  In the returned result, the columns in `ORDER BY` clause are sort key columns.
+
+:::
+
+Since the maximum length of a Prefix index entry is 36 bytes, the exceeded part will be truncated. Therefore, each entry in the Prefix index of this table is uid (4 bytes) + name (only the first 32 bytes are taken), and the prefix fields are `uid` and `name`.
+
+#### Precaution
+
+- The number of prefix fields cannot exceed 3, and the maximum length of a Prefix index entry is 36 bytes.
+
+- Prefix fields support to be of the xxx type.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/ecc35f66-e0f7-48ca-974e-5b3bea303de7)
what "xxx" should be?